### PR TITLE
Feat #49 add edit endpoints for groups

### PR DESCRIPTION
**Pull request by ttayping**
in this branch I created Controller, mapper and payload model for Group, add and update methods.

### CODE REVIEW

File: src/main/java/com/webperside/courseerpbackend/models/payload/group/GroupPayload.java
Comment (gadirovakhadija) [inline src/main/java/com/webperside/courseerpbackend/models/payload/group/GroupPayload.java:1]: we dont need @Getter @Setter in here

File: src/main/java/com/webperside/courseerpbackend/controller/GroupController.java
Comment (gadirovakhadija) [inline src/main/java/com/webperside/courseerpbackend/controller/GroupController.java:1]: You must do all work in Service Layer .You just call service components in your controller class.

File: pom.xml
Comment (gadirovakhadija) [inline pom.xml:1]: please follow codes in main project (for student or subject)

**Review by gadirovakhadija — CHANGES_REQUESTED** [review]

File: src/main/java/com/webperside/courseerpbackend/controller/GroupController.java
Comment (gadirovakhadija) [reply src/main/java/com/webperside/courseerpbackend/controller/GroupController.java:1]: please follow codes in main project (for student or subject)

Comment (ttayping) [reply src/main/java/com/webperside/courseerpbackend/controller/GroupController.java:1]: we get agreement about where mapper procedure must be happen. Final decision is controller class for this case.

File: src/main/java/com/webperside/courseerpbackend/controller/GroupController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/groups")
+@RequiredArgsConstructor
+public class GroupController {
+
+    private final GroupService groupService;
+    private static final GroupEntityMapper groupEntityMapper =GroupEntityMapper.INSTANCE;
```
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/controller/GroupController.java:16]: You don't need to store INSTANCE value of Mapper separately. it is unnecessary. use it directly . f.e
GroupEntityMapper.INSTANCE.toEntity()

**Review by hamidsultanzadeh — CHANGES_REQUESTED** [review]

**Review by hamidsultanzadeh — APPROVED** [review]

**Comment by hamidsultanzadeh** [comment]
@aytiqaqash @gadirovakhadija please leave your decision

**Review by gadirovakhadija — APPROVED** [review]

**Pull request merged**